test/pylib/scylla_server.py

```diff
@@ -337,6 +337,27 @@ async def stop(self) -> None:
             self.control_connection = None
             self.control_cluster = None
 
+    async def stop_gracefully(self) -> None:
```

Comment:
I wonder if we need a timeout param here.

Reply:
Added to TODO list